File: c++/1002.cpp
#include <stdio.h>
#include <math.h>

int main() {
    double pi, raio;

    pi = 3.14159;

    scanf("%lf", & raio);

    printf("A=%.4lf\n", (pi * pow(raio, 2)));

    return 0;
}

File: golang/1002.go
package main

import (
	"fmt"
	"math"
)

func main() {

	const pi = 3.14159

	var n, raio float64

	fmt.Scan(&n)

	raio = pi * math.Pow(n, 2)

	fmt.Printf("A=%.4f\n", raio)

}
